transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringSubmissionQueue.java

```diff
@@ -86,21 +87,34 @@
         this.tail = PlatformDependent.getIntVolatile(kTailAddress);
 
         this.timeoutMemoryAddress = PlatformDependent.allocateMemory(KERNEL_TIMESPEC_SIZE);
+        this.iosqeAsyncThreshold = iosqeAsyncThreshold;
 
         // Zero the whole SQE array first
         PlatformDependent.setMemory(submissionQueueArrayAddress, ringEntries * SQE_SIZE, (byte) 0);
 
         // Fill SQ array indices (1-1 with SQE array) and set nonzero constant SQE fields
         long address = kArrayAddress;
         long sqeFlagsAddress = submissionQueueArrayAddress + SQE_FLAGS_FIELD;
-        byte flag = iosqeAsync ? (byte) Native.IOSQE_ASYNC : 0;
         for (int i = 0; i < ringEntries; i++, address += INT_SIZE, sqeFlagsAddress += SQE_SIZE) {
             PlatformDependent.putInt(address, i);
-            PlatformDependent.putByte(sqeFlagsAddress, flag);
```

Comment:
No need to compute the `sqeFlagsAddress` anymore. The variable isn't used.

Reply:
good catch